### PR TITLE
ruskakimov/call_put_spread

### PR DESCRIPTION
**Pull request by ruskakimov**

### CODE REVIEW

File: src/javascript/app/pages/trade/charts/highchart.ui.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-const addComma = require('../../../common/currency').addComma;
-const localize = require('../../../../_common/localize').localize;
+const addComma      = require('../../../common/currency').addComma;
+const localize      = require('../../../../_common/localize').localize;
```
Comment (negarn) [inline src/javascript/app/pages/trade/charts/highchart.ui.js]: extra whitespace

File: src/javascript/app/pages/trade/tick_trade.js [hunk 2]
```diff
@@ -128,6 +128,8 @@ const TickDisplay = (() => {
                 width          : config.width || (config.minimize ? 394 : null),
                 height         : config.minimize ? 143 : null,
                 backgroundColor: null,
+                // TODO: we pass UNDEFINED as load event callback,
+                // plot() is executed once
```
Comment (negarn) [inline src/javascript/app/pages/trade/tick_trade.js]: what is this comment?

Comment (ruskakimov) [reply src/javascript/app/pages/trade/tick_trade.js]: @negarn on this line
```js
events         : { load: plot(config.plot_from, config.plot_to) },
```
we pass the result of `plot(config.plot_from, config.plot_to)`, which is `undefined` in this case,
this is probably not intentional and what was meant is something like this:
```js
events         : { load: () => {plot(config.plot_from, config.plot_to)} },
```

Comment (negarn) [reply src/javascript/app/pages/trade/tick_trade.js]: I see, then could we fix it here? or it breaks something?

**Comment by ruskakimov** [comment]
some github issues, closing and re-opening a new one

**Pull request closed without merging**